SolWatch/Planet.cs

```diff
@@ -15,13 +15,20 @@ public class Planet
         float meanMotion { get => MathF.Sqrt(Utilities.GravitationalParameter_Sol / MathF.Pow(SemiMajorAxis, 3)); } // radians/s
         float period { get => Utilities.FullCircle / meanMotion; } // s
 
-        public Planet(string name, float semiMajorAxis, float longitudeOfAscendingNode, float argumentOfPeriapsis, EpochAnomaly referencePosition)
+        public Planet(string name, float semiMajorAxis, float longitudeOfAscendingNode, float argumentOfPeriapsis, EpochAnomaly referenceEpochAnomaly, bool anglesInDegrees = false)
         {
+            if (anglesInDegrees)
+            {
+                longitudeOfAscendingNode = Utilities.RadiansFromDegrees(longitudeOfAscendingNode);
+                argumentOfPeriapsis = Utilities.RadiansFromDegrees(argumentOfPeriapsis);
+                referenceEpochAnomaly = new EpochAnomaly(referenceEpochAnomaly.Epoch, Utilities.RadiansFromDegrees(this.referenceEpochAnomaly.Anomaly));
```

Comment:
this.referenceEpochAnomaly.Anomaly should just become referenceEpochAnomaly.Anomaly